cluster/gce/gci/configure-helper.sh

```diff
@@ -375,6 +376,30 @@ current-context: service-account-context
 EOF
 }
 
+function create-kubecontrollermanager-kubeconfig {
+  echo "Creating kube-controller-manager kubeconfig file"
+  mkdir -p /etc/srv/kubernetes/kube-controller-manager
+  cat <<EOF >/etc/srv/kubernetes/kube-controller-manager/kubeconfig
+apiVersion: v1
+kind: Config
+users:
+- name: kube-controller-manager
+  user:
+    token: kube-controller-manager-token
+clusters:
+- name: local
+  cluster:
+    insecure-skip-tls-verify: true
```

Comment:
I pulled out the CA.crt bit when I was thrashing around.  @cjcullen are we ok with this for partial progress or is it easy to fix?